src/solidlsp/language_servers/devsense_php.py

```diff
@@ -0,0 +1,143 @@
+"""
+Provides PHP specific instantiation of the LanguageServer class using Devsense.
+"""
+
+import logging
+import os
+import shutil
+import subprocess
+
+from overrides import override
+
+from solidlsp.ls import SolidLanguageServer
+from solidlsp.ls_config import LanguageServerConfig
+from solidlsp.ls_logger import LanguageServerLogger
+from solidlsp.ls_utils import PlatformId, PlatformUtils
+from solidlsp.lsp_protocol_handler.server import ProcessLaunchInfo
+from solidlsp.settings import SolidLSPSettings
+
+from .common import RuntimeDependency, RuntimeDependencyCollection
+
+
+class DevsensePHP(SolidLanguageServer):
```

Comment:
Thanks for the contribution. _start_server is missing, did you test that this works? You can test manually by adjusting the php tests to use devsense and then running `pytest -m php`.

I suspect this code was never executed since the class can't even be instantiated without an error